3_javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,87 @@
+console.log("Javascript/week1");
+//Age-ify (A future age calculator)
+const yearOfBirth = 1992;
+const yearFuture = 2024;
+const age = yearFuture - yearOfBirth;
+console.log(age);
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+console.log(dogYear);
+const dogYearsCompareToHuman = dogYear * 7;
+console.log(dogYearsCompareToHuman);
+function dogsAge(shouldShowResultInDogYears) {
+    if (shouldShowResultInDogYears) {
+        return console.log("Your dog will be " + dogYear + " human years old in " + dogYearFuture);
+    }
+        return console.log("Your dog will be " + dogYearsCompareToHuman + " dog years old in " + dogYearFuture); 
+}
+
+dogsAge(true);
+dogsAge(false);
+
+//Housey pricey (A house price estimator)
+
+//House for Peter
+const housePriceForPeter = 2500000;
+let houseWideForPeter = 8;
+let houseDeepForPeter = 10;
+let houseHighForPeter =10;
+let houseGardenSizeForPeter = 100;
```

Comment:
these are remain constant as well, so you can replace `let` here with `const` also :)